modules/ROOT/partials/configuration/exportword_watermark.adoc

```diff
@@ -0,0 +1,23 @@
+[[watermark]]
+== `watermark`
```

Comment:
Suggest indenting this as its a option for `exportword_converter_options` and currently it looks like its seperate config option.
```suggestion
=== `watermark`
```